freetextresponse/tests.py

```diff
@@ -260,6 +261,23 @@ def test_compute_score(self, credit):
             {'value': credit.value, 'max_value': Credit.full.value},
         )
 
+    @ddt.data(Credit.zero, Credit.half, Credit.full)
+    def test_compute_score_error(self, credit):
+        # pylint: disable=protected-access
+        """
+        Tests _compute_score's error handling
```

Comment:
The current test only checks for the IntegrityError exception. The try - except that I introduced also only handles the IntegrityError exception. I could have it catch and ignore all exceptions, but I don't know if that is a good idea.